src/duration.ts

```diff
@@ -1,6 +1,7 @@
 import DurationFormat from './duration-format-ponyfill.js'
 import type {DurationFormatOptions} from './duration-format-ponyfill.js'
 const durationRe = /^[-+]?P(?:(\d+)Y)?(?:(\d+)M)?(?:(\d+)W)?(?:(\d+)D)?(?:T(?:(\d+)H)?(?:(\d+)M)?(?:(\d+)S)?)?$/
+const durationRoundingThresholds = [Infinity, 11, 28, 21, 55, 55, 900]
```

Comment:
I'm a little concerned that this array is a level of indirection which makes it hard to figure out the rounding rules. At minimum it might be worth adding comments to each of these explaining that they're months, days, hours and so-on.

Reply:
I just attempted to place the constants at the same place that others appear to be placed, which is to the top of the source file. If putting them next to the function that uses them is better then I will happily do that, in fact I also do prefer it that way. I will be sure to add comments.